### PR TITLE
DSC minor bug fixes

### PR DESCRIPTION
**Pull request by eshaparmar**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@eshaparmar__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (esparmar). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: src/ResourceManager/Compute/Commands.Compute.Test/ScenarioTests/DscExtensionTests.ps1 [hunk 1]
```diff
@@ -105,8 +105,11 @@ function Test-GetAzureRmVMDscExtension
     }
     finally
     {
-        # Cleanup
-        Clean-ResourceGroup $rgname
+		# Cleanup
```
Comment (hovsepm) [inline src/ResourceManager/Compute/Commands.Compute.Test/ScenarioTests/DscExtensionTests.ps1]: please align your code. Replace tab with 4 spaces.

Comment (eshaparmar) [inline src/ResourceManager/Compute/Commands.Compute.Test/ScenarioTests/DscExtensionTests.ps1]: Thanks! fixed it

**Comment by eshaparmar** [comment]
Is this expected to fail? I didnt change anything in the test except the version 

http://azuresdkci.cloudapp.net/job/powershell/3166/artifact/src/Publish/TestResults/Microsoft.Azure.Commands.Compute.Test.html

**Comment by markcowl** [comment]
@eshaparmar Please retarget this at the release branch (release-1.0.0) to make it in for this release.

**Comment by markcowl** [comment]
@eshaparmar closing in favor of the release PR

**Pull request closed without merging**